test/Utils.hs

```diff
@@ -55,13 +55,15 @@ shouldBeSuccess result@(_code, _stdout, _stderr) = do
   result `shouldSatisfy` (\(code, _, _) -> code == ExitSuccess)
 
 shouldBeSuccessOutput :: HasCallStack => FilePath -> (ExitCode, Text, Text) -> IO ()
-shouldBeSuccessOutput expected result = do
-  expectedContent <- readFixture expected
-  result `shouldSatisfy` (\(code, stdout, _stderr) -> code == ExitSuccess && stdout == expectedContent)
+shouldBeSuccessOutput expected (code, stdout, _stderr) = do
+  expectedStdout <- readFixture expected
+  code `shouldBe` ExitSuccess
```

Comment:
I was struggling when fixing the tests, because the assertion error wasn't specific enough.
It was saying something like "Predicate failed blabla" from which it wasn't clear why it failed.
With this change you get more precise string diff about how exactly the stdout differed.

Reply:
Looks great! 👍